### PR TITLE
fix scrollview enable event handling

### PR DESCRIPTION
**Pull request by zilongshanren**
Re: cocos-creator/fireball#4111

Changes proposed in this pull request:
- fix scrollview enable event handling

@cocos-creator/engine-admins

### CODE REVIEW

File: cocos2d/core/components/CCScrollView.js [hunk 1]
```diff
@@ -698,6 +698,8 @@ var ScrollView = cc.Class({
     },
 
     _onMouseWheel: function(event) {
+        if (!this.enabledInHierarchy) return;
```
Comment (jareguo) [inline cocos2d/core/components/CCScrollView.js:701]: 不需要 stopPropagation？

Comment (zilongshanren) [reply cocos2d/core/components/CCScrollView.js:701]: 既然 disable 了，就不用把下面节点的事件给阻断了吧？

Comment (jareguo) [reply cocos2d/core/components/CCScrollView.js:701]: 嗯，是的

**Comment by jareguo** [comment]
LGTM

**Pull request merged**